extensions/github/package.json

```diff
@@ -25,6 +25,9 @@
       "supported": true
     }
   },
+  "enabledApiProposals": [
+	"contribShareMenu"
```

Comment:
Missing formatting?

Reply:
I never realized we use spaces in our json files.